sh/list_storage_and_network_driver_names.sh

```diff
@@ -3,4 +3,4 @@
 BLACKLIST='-e bfa -e csiostor -e cxgb4 -e bna -e aic94xx'
 KERNEL_VERSION=$1
 set -e
-find /lib/modules/$KERNEL_VERSION/kernel/drivers/net/ethernet /lib/modules/$KERNEL_VERSION/kernel/drivers/scsi -type f -printf '%f\n' | sed 's/\.ko$//' | grep -v $BLACKLIST
+find /lib/modules/$KERNEL_VERSION/kernel/drivers/net/ethernet /lib/modules/$KERNEL_VERSION/kernel/drivers/scsi /lib/modules/$KERNEL_VERSION/kernel/drivers/nvme /lib/modules/$KERNEL_VERSION/kernel/drivers/message -type f -printf '%f\n' | sed 's/\.ko$//' | sed 's/\.ko.xz$//' | grep -v $BLACKLIST
```

Comment:
@susyo what is the reason for this change? Seems legit jsut want to understand why

Reply:
adding more storage driverss and network drivers  that are not in the above paths

Reply:
adding more storage and network drivers that are not in the above paths